src/flightlog_fielddefs.js

```diff
@@ -134,6 +134,49 @@ export const FLIGHT_LOG_FLIGHT_MODE_NAME_POST_3_3 = makeReadOnly([
   "LAUNCHCONTROL",
 ]);
 
+export const FLIGHT_LOG_FLIGHT_MODE_NAME_POST_4_6 = makeReadOnly([
```

Comment:
```suggestion
export const FLIGHT_LOG_FLIGHT_MODE_NAME_POST_4_5 = makeReadOnly([
```

Reply:
I understand: the key word POST. Can be 4_5. Ok.